manifests/c/CWEcomputerservices/GSAK/9.0.0.14/CWEcomputerservices.GSAK.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: CWEcomputerservices.GSAK
+PackageVersion: 9.0.0.14
+PackageLocale: en-US
+Publisher: CWE computer services
+PackageName: GSAK - Geocaching Swiss Army Knife

```

Comment:
```suggestion
PackageName: GSAK
```
![image](https://user-images.githubusercontent.com/11600822/120901863-2cac7c80-c635-11eb-9f8c-3a62bbc154fb.png)

Reply:
Thanks a lot for your help!